opentelemetry-sdk/src/opentelemetry/sdk/environment_variables/__init__.py

```diff
@@ -98,6 +98,13 @@
 Default: 512
 """
 
+OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT = "OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT"
+"""
+.. envvar:: OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT
+
+The :envvar:`OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT` represents the maximum allowed attribute length.
+"""
+
 OTEL_EVENT_ATTRIBUTE_COUNT_LIMIT = "OTEL_EVENT_ATTRIBUTE_COUNT_LIMIT"
 """
```

Comment:
Could you modify the docstring for `OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT` to say that it is specific for attributes on span? AS well, it takes precedence over `OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT` specifically for span attributes.